Modules/itertoolsmodule.c

```diff
@@ -3454,12 +3454,83 @@ count_repr(countobject *lz)
                                 lz->long_cnt, lz->long_step);
 }
 
+static PyObject *
+count_peek(countobject *lz, PyObject *Py_UNUSED(ignored))
+{
+    PyObject *long_cnt = lz->long_cnt;
+    if (long_cnt != NULL) {
+        Py_INCREF(long_cnt);
+        return long_cnt;
```

Comment:
```suggestion
		return Py_NewRef(long_cnt);
```

And no, you cannot rerun tests without a push.